### PR TITLE
fix(ci): increase log level for unit test, fix connector node ci

### PR DESCRIPTION
**Pull request by WillyKidd**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- fixed an error in a unit test https://github.com/risingwavelabs/risingwave/pull/9575
- increased connector unit test log level to ERROR (less INFO logs...)

## Checklist For Contributors

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
~~- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).~~
~~- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)~~
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [x] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

### CODE REVIEW

**Comment by StrikeW** [comment]
IMO we can improve the organization of test cases by collecting all unit tests in a dedicated module and grouping different tests by package. Then all unit tests can share some common configuration. @WillyKidd

File: .github/workflows/connector-node-integration.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         java: [ '11', '17' ]
-    name: Java ${{ matrix.java }}
```
Comment (StrikeW) [inline .github/workflows/connector-node-integration.yml:17]: Seems the CI pipeline cannot detect the new name, let's revert this line and see the result.

File: java/connector-node/risingwave-source-test/src/test/java/com/risingwave/connector/MySQLSourceTest.java [hunk 2]
```diff
@@ -166,7 +166,7 @@ public void testPermissionCheck() throws SQLException {
         Connection connDbz = SourceTestClient.connect(dbzDataSource);
```
Comment (StrikeW) [inline java/connector-node/risingwave-source-test/src/test/java/com/risingwave/connector/MySQLSourceTest.java:166]: Why this connection is unused?

File: .github/workflows/connector-node-integration.yml (hunk 1 above)
Comment (WillyKidd) [reply .github/workflows/connector-node-integration.yml]: reverted ;)

**Review by StrikeW — APPROVED** [review]

**Pull request merged**